README.md

```diff
@@ -75,6 +75,17 @@ Then click "Add More Scanning Tools."
 
 ![image](/images/install04.png)
 
+### Workflow Setup - Organization Wide
+The scorecard workflow can be automatically added to every repository under an organization using the [multi-repo-action](https://github.com/ossf/scorecard-action/tree/main/multi-repo-action) tool.
+1) `git clone ` this repository & `cd multi-repo-action`
```

Comment:
Specify which repository you mean by "this repository" -- i.e., `git clone the cd multi-repo-action repository and this repository (scorecard-action)`